enclave_manager/Dockerfile

```diff
@@ -171,31 +164,53 @@ RUN mkdir -p /opt/intel \
  && mkdir -p /etc/init
 WORKDIR /opt/intel
 
-# Add Intel SGX repo to apt sources and install Intel SGX PSW packages
-RUN echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main' | tee /etc/apt/sources.list.d/intel-sgx.list \
- && wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add \
- && apt-get update \
- && apt-get install -y -q \
-    libsgx-enclave-common \
-    libsgx-launch \
-    libsgx-urts \
-    libsgx-epid \
-    libsgx-quote-ex \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/*
-
-RUN SGX_SDK_FILE=sgx_linux_x64_sdk_2.10.100.2.bin \
-  && wget https://download.01.org/intel-sgx/sgx-linux/2.10/distro/ubuntu18.04-server/$SGX_SDK_FILE \
-  && echo "yes" | bash ./$SGX_SDK_FILE \
-  && rm $SGX_SDK_FILE \
-  && echo ". /opt/intel/sgxsdk/environment" >> /etc/environment
-
+RUN if [ "$DISTRO" = "bionic" ] ; then \
+      # Add Intel SGX repo to apt sources and install Intel SGX PSW packages
+      echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main' | tee /etc/apt/sources.list.d/intel-sgx.list \
+      && wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add \
+      && apt-get update \
+      && apt-get install -y -q \
+         libsgx-enclave-common \
+         libsgx-launch \
+         libsgx-urts \
+         libsgx-epid \
+         libsgx-quote-ex \
+      && apt-get clean \
+      && rm -rf /var/lib/apt/lists/* \
+     
+      && SGX_SDK_FILE=sgx_linux_x64_sdk_2.10.100.2.bin \
+      && wget https://download.01.org/intel-sgx/sgx-linux/2.10/distro/ubuntu18.04-server/$SGX_SDK_FILE \
+      && echo "yes" | bash ./$SGX_SDK_FILE \
+      && rm $SGX_SDK_FILE \
+      && echo ". /opt/intel/sgxsdk/environment" >> /etc/environment; \
+
+    elif [ "$DISTRO" = "centos" ] ; then \
+      wget https://download.01.org/intel-sgx/sgx-linux/2.10/distro/centos8.1-server/sgx_rpm_local_repo.tgz \
+      && tar -xvf sgx_rpm_local_repo.tgz \
+      && yum-config-manager --add-repo file:///opt/intel/sgx_rpm_local_repo \
+      && cd sgx_rpm_local_repo \
```

Comment:
Do we need `cd sgx_rpm_local_repo`  since we already executed `yum-config-manager --add-repo file:///opt/intel/sgx_rpm_local_repo `

Reply:
not needed...removed it now.